retis/src/process/cli/sort.rs

```diff
@@ -72,7 +72,7 @@ impl SubCommandParserRunner for Sort {
         let mut factory = FileEventsFactory::new(self.input.as_path())?;
 
         if matches!(factory.file_type(), file::FileType::Series) {
-            println!("File already sorted");
+            log::info!("File already sorted");
```

Comment:
If we run without logs, or with suppressed stderr we won't see anything. Wouldn't that be confusing?

Reply:
In case `-o` is used no, in case stdout is you're right. However all error / info messages are handles by `log` (we even remove the level from the output from `info!` messages are they're part of the expected output), the above was for consistency. If the user explicitly removes those messages then it's expected to have less explanations in regards to what is happening.

No strong feeling though, I can remove it if you prefer.

Reply:
> Another option is to use `log:info!` in case `-o` is used and `bail!` otherwise.


Why not bail in both cases?

Reply:
I'm not sure if I like this conditional error. I prefer just using `log` as it is in the patch.

> We could improve things to make `sort` only sorting a file and `print` to have a `--sort` option. Not 100% sure about that though.

Me neither.